server/sonar-server/src/test/java/org/sonar/server/test/ws/ListActionTest.java

```diff
@@ -31,280 +31,255 @@
 import org.sonar.db.DbClient;
 import org.sonar.db.DbTester;
 import org.sonar.db.component.ComponentDto;
+import org.sonar.db.protobuf.DbFileSources;
+import org.sonar.db.source.FileSourceDto;
 import org.sonar.server.component.ComponentFinder;
 import org.sonar.server.es.EsTester;
 import org.sonar.server.exceptions.ForbiddenException;
 import org.sonar.server.exceptions.NotFoundException;
-import org.sonar.server.test.index.CoveredFileDoc;
-import org.sonar.server.test.index.TestDoc;
 import org.sonar.server.test.index.TestIndex;
 import org.sonar.server.test.index.TestIndexDefinition;
+import org.sonar.server.test.index.TestIndexer;
 import org.sonar.server.tester.UserSessionRule;
-import org.sonar.server.ws.WsTester;
+import org.sonar.server.ws.TestRequest;
+import org.sonar.server.ws.WsActionTester;
+import org.sonarqube.ws.MediaTypes;
+import org.sonarqube.ws.WsTests;
+import org.sonarqube.ws.WsTests.ListResponse;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.sonar.api.resources.Qualifiers.UNIT_TEST_FILE;
+import static org.sonar.api.web.UserRole.CODEVIEWER;
+import static org.sonar.api.web.UserRole.USER;
+import static org.sonar.db.component.ComponentTesting.newFileDto;
+import static org.sonar.db.component.ComponentTesting.newProjectDto;
+import static org.sonar.db.protobuf.DbFileSources.Test.TestStatus.OK;
+import static org.sonar.server.test.db.TestTesting.newTest;
+import static org.sonar.server.test.ws.ListAction.SOURCE_FILE_ID;
+import static org.sonar.server.test.ws.ListAction.SOURCE_FILE_KEY;
+import static org.sonar.server.test.ws.ListAction.SOURCE_FILE_LINE_NUMBER;
+import static org.sonar.server.test.ws.ListAction.TEST_FILE_ID;
+import static org.sonar.server.test.ws.ListAction.TEST_FILE_KEY;
+import static org.sonar.server.test.ws.ListAction.TEST_ID;
 
 public class ListActionTest {
 
-  private static final String DEFAULT_ORGANIZATION_UUID = "org1";
-
-  @Rule
-  public EsTester es = new EsTester(new TestIndexDefinition(new MapSettings()));
   @Rule
   public ExpectedException expectedException = ExpectedException.none();
   @Rule
+  public EsTester es = new EsTester(new TestIndexDefinition(new MapSettings()));
+  @Rule
   public UserSessionRule userSessionRule = UserSessionRule.standalone();
   @Rule
-  public DbTester db = DbTester.create(System2.INSTANCE)
-      .setDefaultOrganizationUuid(DEFAULT_ORGANIZATION_UUID);
+  public DbTester db = DbTester.create(System2.INSTANCE);
```

Comment:
could you do the same thing for the constructor `DbTester.create()` if it's not already done?